cluster-scope/base/core/namespaces/physrisk-resilience/kustomization.yaml

```diff
@@ -1,9 +1,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- namespace.yaml
+  - namespace.yaml
```

Comment:
The `resourcequota.yaml` file needs to be listed here, otherwise the quota won't apply.

Reply:
Done.